src/Components/CreateImageWizard/utilities/useValidation.tsx

```diff
@@ -166,7 +166,10 @@ export function useFilesystemValidation(): StepValidation {
 
   const duplicates = getDuplicateMountPoints(partitions);
   for (const partition of partitions) {
-    if (!isMountpointMinSizeValid(partition.min_size)) {
+    if (partition.min_size === '') {
+      errors[`min-size-${partition.id}`] = 'partition size is required';
```

Comment:
One tiny nitpick, otherwise looks great. Thank you!

```suggestion
      errors[`min-size-${partition.id}`] = 'Partition size is required';
```